### PR TITLE
[squid] Add metrics to squid

### PR DESCRIPTION
**Pull request by nabeo**
add metrics from mgr:info.

### CODE REVIEW

File: mackerel-plugin-squid/lib/squid.go [hunk 1]
```diff
@@ -42,7 +79,12 @@ func (m SquidPlugin) FetchMetrics() (map[string]interface{}, error) {
 		return nil, err
 	}
 	fmt.Fprintln(conn, "GET cache_object://"+m.Target+"/info HTTP/1.0\n\n")
-	scanner := bufio.NewScanner(conn)
+	return m.ParseMgrInfo(conn)
+}
+
+// ParseMgrInfo parser for squid mgr:info
```
Comment (astj) [inline mackerel-plugin-squid/lib/squid.go]: Can you add links to reference of `/info` ?
(I guess https://wiki.squid-cache.org/Features/CacheManager/Info?highlight=%28Feature..Squid.Cache.Manager%29)

File: mackerel-plugin-squid/lib/squid_test.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+func TestGraphDefinition(t *testing.T) {
+  var squid SquidPlugin
+
+  graphdef := squid.GraphDefinition()
+  if len(graphdef) != 6 {
+    t.Errorf("GetTempfilename: %d should be 6", len(graphdef))
+  }
+}
+
+func TestParse(t *testing.T) {
```
Comment (astj) [inline mackerel-plugin-squid/lib/squid_test.go]: :+1:

**Comment by astj** [comment]
@nabeo 
Thank you!
Changes seems good, but I'd like to request to add links for mgr:info.

File: mackerel-plugin-squid/lib/squid.go (hunk 1 above)
Comment (nabeo) [reply mackerel-plugin-squid/lib/squid.go]: add url link at b1141bba7a99c1c8b68166f291f920556d8bcf40

**Comment by astj** [comment]
lgtm! Thank you.

**Pull request merged**